template/metro.config.js

```diff
@@ -5,7 +5,22 @@
  * @format
  */
 
+const INTERNAL_CALLSITES_REGEX = new RegExp(
```

Comment:
Should we be hiding this by default but let people override it here?

Reply:
Yes. There was some discussion on the RN Discord but I didn't get around to making the changes today. The basic idea is to put the defaults in the CLI, where there's precedent for this kind of thing.